### ISSUE TITLE
Error in rule generate_full_probes

### ISSUE THREAD

**Issue by MiriJ15** [proudquartz/hiprfish#3]
Hello,
I have been following your guide, but I always enup with the same error that prevents me from getting summary containing full probes. I am using exactly the same setup, and can view all probes in simulation folder (but there are no blocker probes). Have you faced this problem when you were testing the code?

![image_2022-04-01_004558](https://user-images.githubusercontent.com/86243044/161196723-57ccf97e-8070-4439-90a1-f2612328d2d0.png)

**Comment by proudquartz** [comment]
I have not seen this before. I would suggest you run the relevant blocks of
code line by line. It seems like pandas is not happy with the output format
of the aggregation functions.

**Comment by MiriJ15** [comment]
Thank you, its working right now. I had to make few changes in order to make code work.
1. I had to install primer3 separately and change the path in config from 'primer3_core' to full path to .exe file.
2. I had an issue with pandas, due to major update being released in February and it was causing millions of warning outputs in terminal and error in rule generate_full_probes. My solution was to uninstall pandas installed via Miniconda, because it is installing latest packages by default and use "conda install pandas=1.3.1" command instead.
